presto-docs/src/main/sphinx/release/release-0.290.rst

```diff
@@ -0,0 +1,109 @@
+* Add support in QueuedStatement protocol to accept pre-minted query id and slug :pr:`23407`.
+* Add support to proxy AuthorizedIdentity via JWT :pr:`23546`.
+* Added support for casting char datatype to various numeric datatypes.     :pr:`23792`.
+* Adds a new NodeManager : 'PluginNodeManager' :pr:`23863`.
+* Replace configuration property `async-cache-full-persistence-interval` with `async-cache-persistence-interval`. :pr:`23626`.
+* Remove array_dupes and array_has_dupes alias names from functions :func:`array_duplicates` and :func:`array_has_duplicates`.
+* Avoid pushdown of negative position for element_at for array  :pr:`23479`.
+* GET `/v1/info/state` to return INACTIVE state until the resource group configuration manager is fully initialized :pr:`23585`.
+* Make IcebergDistributedSmokeTestBase abstract :pr:`23580`.
+* Upgraded avro to version 1.11.4 :pr:`23868`.
+* Upgraded commons-codec to version 1.17.0 :pr:`23868`.
+* Upgraded commons-compress to version 1.26.2 :pr:`23868`.
+* Upgraded commons-io to version 2.16.0 :pr:`23794`.
+* Upgraded commons-io to version 2.16.1 :pr:`23868`.
+* Upgraded commons-lang3 to version 3.14.0 :pr:`23868`.
+* Upgraded protobuf-java to version 3.25.5 :pr:`23797`.
+* Upgraded protobuf-java-util to version 3.25.5 :pr:`23797`.
+
+Security Changes
+________________
+* Upgrade Postgres JDBC Driver to 42.6.1 :pr:`23710`.
+
+Hive Connector Changes
+______________________
+* Fix interpretation of ambiguous timestamps inside array, map, or row types for tables using TEXTFILE format to interpret the timestamps as the earliest possible unixtime for consistency with the rest of Presto.
+* Fix timestamps inside array, map, or row types for tables using TEXTFILE format to respect the ``hive.time-zone property``.
+
+Iceberg Connector Changes
+_________________________
+* Fix time-type columns to return properly when ``iceberg.parquet-batch-read-optimization-enabled`` is set to ``TRUE``. :pr:`23542`.
+* Fix to reduce drop time for Iceberg tables with deleted metadata in S3 storage. :pr:`23510`.
+* Add Iceberg metadata table $ref :pr:`23503`.
+* Add configuration property ``iceberg.rest.auth.oauth2.scope`` for OAUTH2 authentication in Iceberg's REST catalog :pr:`23884`.
+* Add iceberg.rest.auth.oauth2.uri configurable property :pr:`23739`.
+* Add procedure `rollback_to_timestamp` to rollback an iceberg table to a given point in time. :pr:`23559`.
+* Add support of UUID-typed columns :pr:`23627`.
+* Add support to query Iceberg table by branch/tag name :pr:`23539`.
+* Add table property `metrics_max_inferred_column` to configure the max columns number for which metrics are collected, and support `metrics_max_inferred_column` for Iceberg table with `PARQUET` format :pr:`23468`.
+* Support procedure fast_forward for iceberg :pr:`23589`.
+* Support timestamp without timezone in time travel expressions :pr:`23714`.
+* Support using named arguments in procedure `register_table` and `unregister_table` :pr:`12345`.
```

Comment:
```suggestion
* Add support for using named arguments in procedure `register_table` and `unregister_table` :pr:`23592`.
```
Rephrased to follow the phrasing in the [Order of changes](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines#order-of-changes) in the Release Notes Guidelines. Please review my rephrasing for accuracy.

Rephrasing may mean this entry should be moved to follow the sequence in the [Order of changes](https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines#order-of-changes).